src/App.js

```diff
@@ -14,4 +22,6 @@ class App extends Component {
     }
 }
 
-export default App;
+export default connect.defaults({fetch: ConnectionComponent.switchFetch})(props => ({
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "props". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)